service.yml

```diff
@@ -19,6 +19,7 @@ sonarqube:
     - javascript
     - typescript
   enable: true
+  main_branch: main
```

Comment:
This will only take effect after the PR is merged (and servicebot is run for this repo).